gsuite_activityevent_rules/gsuite_brute_force_login.yml

```diff
@@ -0,0 +1,58 @@
+AnalysisType: rule
+Filename: gsuite_brute_force_login.py
+RuleID: GSuite.BruteForceLogin
+DisplayName: --DEPRECATED-- GSuite Brute Force Login
+Enabled: false
+LogTypes:
+  - GSuite.ActivityEvent
```

Comment:
Reasonably sure this is right? Do we have a listing of the expected types somewhere?